common/test/acceptance/pages/studio/container.py

```diff
@@ -191,10 +193,19 @@ def toggle_staff_lock(self, inherits_staff_lock=False):
         Returns True if the lock is now enabled, else False.
         """
         was_locked_initially = self.is_staff_locked
+
+        # Sections and Units lock differently than subsections, so try both actions
```

Comment:
I was wondering why we have to care about doing this check in 2 different files when I realized - this isn't the modal at all. It's the publish view of a single unit. My changes related to subsection content hiding don't apply here at all.